core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -932,9 +938,43 @@ public static String getDefaultOncoprintView() {
         }
         return defaultOncoprintView.trim();
     }
-
+    
+    public static boolean useCustomDriverAnnotations() {
+        String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS);
```

Comment:
In `portal.properties.EXAMPLE` the custom Driver Annotations are set to `false`. So do we want only to set this parameter to `true` when the property is empty, or also "by default"? (as stated in `portal.properties.EXAMPLE`)